.github/workflows/stale.yml

```diff
@@ -1,18 +1,51 @@
-name: Mark and close stale pull requests
+# This workflow alerts and then closes the stale issues/PRs after specific time
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions/stale
+
+
+name: Mark and close stale PRs/issues
 
 on:
   schedule:
   - cron: "30 1 * * *"
+  
+
+permissions:
+  contents: read
+  issues: write
+  pull-requests: write
 
 jobs:
   stale:
 
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/stale@v3.0.8
+    - uses: actions/stale@v7
       with:
         repo-token: ${{ secrets.GITHUB_TOKEN }}
         stale-pr-message: 'This PR is stale because it has been open 30 days with no activity. Remove stale label or comment or this will be closed in 5 days'
         days-before-stale: 30
         days-before-close: 5
+        
+        # comment on issues if not active for more then 14 days.
+        stale-issue-message: 'This issue has been automatically marked as stale because it has no recent activity. It will be closed if no further activity occurs. Thank you.'
```

Comment:
'This issue has been marked stale because it has no recent activity since 14 days. It will be closed if no further activity occurs. Thank you.'